config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  get 'messages/random_greeting'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  Rails.application.routes.draw do
```

Comment:
### Optional
- [ ] It would be better if we don't repeat ` Rails.application.routes.draw` as it has been defined already, no need to have it again on line no. 7